### PR TITLE
hwo to create a video clarify

### PR DESCRIPTION
**Pull request by noxonsu**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -64,9 +64,14 @@ https://github.com/noxonsu/tools.onout.org/blob/main/contributing.md - наш г
 
 # шаг 4 пререлиз
 1. добавить в dash.onout.org в раздел presale . https://github.com/noxonsu/dash.onout.org/blob/main/src/constants/index.ts#L130 (скопировать данные, самое главное id чтоб был различный от других продуктов)
-2. записать видос (вы записываете на английском). Как на своем домене зайти в админку, задеплоить контракты, как отобразить интерфейс на домене. Второе видео как юзер пользуется этой апкой
 
-6. отправить все материалы нам на ревью, далее фокус группе 
+## 2. записать видос (вы записываете на английском). 
+### 2.1 Первое видео как юзер пользуется этой апкой. не забыть логотип поменять на наш дефолтный "your logo here"  
+Видос залить в в наш канал https://www.youtube.com/channel/UCEu5-QDPFCxKm9z4BAYQSJw (логин support@oout.org). Создать новый плейлист.
```
Comment (VitaliyShulik) [inline contributing.md:70]: support@onout.org

**Review by VitaliyShulik — APPROVED** [review]

File: contributing.md [hunk 2]
```diff
@@ -64,9 +64,57 @@ https://github.com/noxonsu/tools.onout.org/blob/main/contributing.md - наш г
...
+https://support.onout.org/chat/widget/form/3882717100 - Web chat
+https://t.me/onoutsupportbot - Telegram Bot Chat
+https://support.onout.org/hc/1331700057 - Knowledge Base
+
+Join the community! 
+https://discord.gg/VwKEmHEgVN  - Join Discord
+https://dash.onout.org/ - Join mailing list
+https://twitter.com/onout_org - Twitter
+```
+название видео "How to create a DEX without codeing skills at your own domain. Uniswap v2 fork (copy) tutorial". названия меняем. обязательно как можно больше ключей (см семантика п. 1)
```
Comment (jsohex) [inline contributing.md:113]:
```suggestion
Название видео: "How to create a DEX on your own domain without coding skills. Uniswap v2 fork (copy) tutorial". Названия меняем в соответствии со своим продуктом. 
Обязательно использовать как можно больше ключевых слов (см семантика п. 1)
```

File: contributing.md [hunk 3]
```diff
@@ -64,9 +64,57 @@ https://github.com/noxonsu/tools.onout.org/blob/main/contributing.md - наш г
 
 # шаг 4 пререлиз
 1. добавить в dash.onout.org в раздел presale . https://github.com/noxonsu/dash.onout.org/blob/main/src/constants/index.ts#L130 (скопировать данные, самое главное id чтоб был различный от других продуктов)
-2. записать видос (вы записываете на английском). Как на своем домене зайти в админку, задеплоить контракты, как отобразить интерфейс на домене. Второе видео как юзер пользуется этой апкой
 
-6. отправить все материалы нам на ревью, далее фокус группе 
+## 2. записать видос (вы записываете на английском).
```
Comment (jsohex) [inline contributing.md:68]:
```suggestion
## 2. Записать видео (на английском).
Вы записываете либо своим голосом (желательно без слов паразитов и длинных пауз), либо используете любые ресурсы и инструменты для синтезирования искусственного голоса максимально приближенный к реальному.
```

File: contributing.md [hunk 4]
```diff
@@ -64,9 +64,57 @@ https://github.com/noxonsu/tools.onout.org/blob/main/contributing.md - наш г
...
+Join the community! 
+https://discord.gg/VwKEmHEgVN  - Join Discord
+https://dash.onout.org/ - Join mailing list
+https://twitter.com/onout_org - Twitter
+```
+название видео "How to create a DEX without codeing skills at your own domain. Uniswap v2 fork (copy) tutorial". названия меняем. обязательно как можно больше ключей (см семантика п. 1)
+
+Если админка еще не готова, то делаем админку, а этот видос записываем уже когда выходит финальный релиз.
+
+Ссылки на видосы скинуть в дискорд в канал соотсвествующий и опубликовать в твитере. Текст "Hello, we have done 50% of X fork. See videos. From the users side: (link). From the admin's side: (link) . This product is added to "presale" tab on dash.onout.org . Presale price is 700$. Estimate time to release - 1 month. Cheers!"
```
Comment (jsohex) [inline contributing.md:117]:
```suggestion
Ссылки на видео нужно скинуть в дискорд в соотвествующий канал и опубликовать в твиттере. 
Текст "Hello, we have done 50% of development progress a {название продукта} fork. Watch videos: the user usage (link) and the admin usage (link) . This product added to "presale" tab on dash.onout.org. Presale price - $700. Estimate time for releasing is around 1 month. Cheers!"
```

**Review by jsohex — CHANGES_REQUESTED** [review]

**Pull request merged**